### ISSUE TITLE
Delete account crash

### ISSUE THREAD

**Issue by hannesa2** [thunderbird/thunderbird-android#4697]
I'm on current master 
<img width="961" alt="image" src="https://user-images.githubusercontent.com/3314607/80302350-d4584980-87a9-11ea-9c86-0e428284a8ee.png">

and after delete an account I run into this

<img width="869" alt="image" src="https://user-images.githubusercontent.com/3314607/80302336-b0950380-87a9-11ea-8302-e2e1374ac447.png">

Now I can start app again, but selection an account I run into next crash

<img width="829" alt="image" src="https://user-images.githubusercontent.com/3314607/80302395-13869a80-87aa-11ea-8009-42ef5ba9a328.png">

**Comment by GNUDimarik** [comment]
Hey.
@hannesa2 please provide steps to reproduce. I'll try to fix that

**Comment by hannesa2** [comment]
`git checkout 447dbe1`

"_and after delete an account I run into this_ "  ... but maybe it's gone on current recent master.

**Comment by poldi171254** [comment]
I've deleted accounts (test2@zudiewiener.com)  on V 5.731 without any issues. I did, however, find an issue.

After deleting the account, the new account look screen (sorry, not sure what else to call it) still shows the deleted account.
After restarting the emulator everything looks and behaves as it should.

![Test2AfterDelete](https://user-images.githubusercontent.com/43848359/115507746-dc18d400-a2bf-11eb-9d32-e28d2483541a.png)
![Test2ShowsAfterDelete](https://user-images.githubusercontent.com/43848359/115507751-dd4a0100-a2bf-11eb-80f8-e237fd7db353.png)
![correct after restart](https://user-images.githubusercontent.com/43848359/115507764-e1761e80-a2bf-11eb-957b-2089bdc456ae.png)
